docker-compose.yml

```diff
@@ -130,6 +130,15 @@ services:
       - "9200:9200"
       - "9300:9300"
 
+  # https://www.docker.elastic.co/
+  elasticsearch7:
+    image: docker.elastic.co/elasticsearch/elasticsearch-oss:7.1.0
+    environment:
+      - discovery.type=single-node
```

Comment:
Their install docs suggest this: https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html